tools/devops/Makefile

```
@@ -0,0 +1,10 @@
+TOP=../..
+include $(TOP)/Make.config
+
+device-tests-provisioning.csx: device-tests-provisioning.csx.in Makefile $(TOP)/Make.config
+	$(Q_GEN) sed \
+		-e 's#@XCODE_XIP_NAME@#$(XCODE_XIP_NAME)#g' \
```

Minor improvement: this would make the separate variable unnecessary:

```suggestion
		-e 's#@XCODE_XIP_NAME@#$(notdir $(XCODE_URL))#g' \
```

---

Actually that's required, otherwise this happens:

```
[2020-05-21T07:39:00.304Z] Checking system...
[2020-05-21T07:39:00.304Z]     Found OSX 10.15.3 (at least 10.15 is required)
[2020-05-21T07:39:00.304Z]         Downloading Xcode 11.5 from http://xamarin-storage/bot-provisioning/xcodes/$(XCODE_XIP_NAME) to /tmp/x-provisioning...
```

---

Fix: #8641.

---

Thanks Rolf!